Frontend/src/pages/admin/AdminCreateNotificationPage.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <BasePage title="Send Notification">
+    <q-card>
+      <div class="q-pa-md">
+        <q-table
+          title="Select Students"
+          :data="tableStudents"
+          :columns="columns"
+          row-key="email"
+          selection="multiple"
+          :selected.sync="toSendTo"
+        />
+        <q-input
+          outlined
+          v-model="title"
+          label="Title"
+          class="notifPageComponents"
+        />
+        <q-input
+          v-model="body"
+          label="Body"
+          outlined
+          type="textarea"
+          class="notifPageComponents"
+        />
+        <q-btn
+          color="primary"
+          @click="sendNotification()"
+          label="Send Notification"
+          class="notifPageComponents"
+        />
```

Comment:
Add a `q-spinner` here that should show while the notifications are being created (just better for UX, it's been done in a bunch of other pages/components already)